SDPi_Supplement/asciidoc/volume1/tf1-ch-10-sdpi-p.adoc

```diff
@@ -12,47 +11,47 @@ The establishment of a trusted ecosystem of medical and non-medical devices and
 
 _This_ is primarily an IHE transport profile footnote:[See the IHE Technical Frameworks General Introduction for a more detailed description of IHE profile types, published at https://profiles.ihe.net/GeneralIntro/[profiles.ihe.net/GeneralIntro/].], although it does define several content modules detailed in IHE Devices TF-3.
 It supports the transactions and information exchanged in accordance to a Service-Oriented Architecture (SOA) specialized for high-acuity points of care (e.g., operating table or ICU bed), defined as a <<term_service_oriented_medical_device_system>>.
-All the SDPi-P actors are therefore scoped with “<<acronym_somds>>” to clearly identify their application context and scope.
+All the SDPi-P Actors are therefore scoped with “<<acronym_somds>>” to clearly identify their application context and scope.
 
 Although all information exchanged between SDPi-P <<acronym_somds>> participating systems and applications must conform to the basic <<acronym_sdc>>/<<acronym_biceps>> content module requirements footnote:[See <<vol3_clause_sdc_biceps_semantic_content_module>>. ], content modules have been defined for common high-acuity medical devices such as infusion pumps, ventilators and physiologic monitors.
```

Comment:
Does this sentence miss a predicate or something else? Maybe my bad understanding of the english language :-)